vcash-on-ubuntu/README.md

```diff
@@ -0,0 +1,3 @@
+# Vcash Node on Ubuntu VM
+
+This Microsoft Azure template deploys a Vcash public node that will connect to the main Vcash decentralised network.
```

Comment:
Should add more information about what Vcash is and links to Github repo, project site, documentation